src/Raven.Studio/typescript/commands/maintenance/monitorRestoreCommand.ts

```diff
@@ -7,7 +7,7 @@ class monitorRestoreCommand extends commandBase {
     }
 
     execute(): JQueryPromise<any> {
-        new getDocumentWithMetadataCommand("Raven/Restore/Status", null)
+        new getDocumentWithMetadataCommand("Raven/Restore/Status", null) //TODO: obsolete
```

Comment:
It isn't a document, but we do need to track it, I think @grisha-kotler is already doing it.